src/runtime/virtcontainers/acrn.go

```diff
@@ -591,6 +591,15 @@ func (a *Acrn) PauseVM(ctx context.Context) error {
 	return nil
 }
 
+func (a *Acrn) AttestVM(ctx context.Context) error {
+	span, _ := katatrace.Trace(ctx, a.Logger(), "AttestVM", acrnTracingTags, map[string]string{"sandbox_id": a.id})
+	defer span.End()
+
+	// Not supported. return success
```

Comment:
- What's the implication of returning success here? Could a user believe attestation was successful when it never happened?
- Why is it not supported? Is there a canonical URL you can provide? If not, you could raise a Kata issue and add the URL here for tracking / visibility of the limitation.
- How about atleast adding a warning/bug log call here if you must return success?